### DISCUSSION TITLE
Can't verify my academic status as Faculty

### DISCUSSION THREAD

**Discussion by igsbma** [community/community#49447] [GitHub Education] [answered]
I have my employment verification letter pdf from the school's human resources. It has school letter head, my name, the issue date (as of today Mar 7, 2023), my faculty title and affiliation, employment since date. That should be good enough for the requirement listed on the verification upload document page.

After converted the pdf to a larger than 1024*768 JPEG, I uploaded the jpeg and submitted the verification request.

However, your system failed to verify my affiliation status. For example, this is one of the many failed requests:
https://education.github.com/discount_requests/9587484

Can anyone help? I have to use employment verification letter since the school faculty id has no date on it.

Regards,
Bing

**Comment by igsbma** [comment]
Tried some other attempts without success.
Example:
https://education.github.com/discount_requests/9590373

**Comment by igsbma** [comment]
@samus-aran 
Based on your announcement, https://github.com/orgs/community/discussions/49366, I can't submit support ticket to https://support.github.com/contact/education. Once I choose Application not approved or Status of Application, the Subject and Description fields grey out.

The only place is here on the community forum discussions. Can anyone on the discussion forum with Github help with this issue?

BTW, I have already filled in my User Profile with my name and my bio as the error message suggested.

Thanks,
Bing

**Reply by samus-aran** [reply]
Hi @igsbma 👋 

Thanks for the mention 👍 
Yes, we're able to assist by providing guidance for users to work through their rejection reasons here on the community.

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

I see you have done this, but I would also suggest to wait between updating the profile and submitting again so that the system can ensure it has the latest changes you've made to your profile. 

Another tip is to ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor rather than a phone perhaps. Try and not have too much space around the evidence and the background, if possible take an image including the most important areas of the evidence to make the text legible. 

Hope this helps. 👍

**Reply by igsbma** [reply]
Thank you for the reply, @samus-aran!

I had a ticket opened two days ago. Yet, nothing is going on except the 'Rest assured your request is in the right hand' auto-generated message.
This is the Ticket ID, 2047163. Not sure if you can take a look and speed up the process?

Much appreciate any help!

Regards,
Bing

**Comment by igsbma** [comment]
@SayedAlAhlMohamed

Seems you are replying to several related questions. Can you help?
To summary:
1. I have already filled in User Profile with Full Nam, Bio, any linkable website to my affiliation
2. I have University Human Resource issued Employment Verification Letter uploaded.

Thanks,
Bing

**Comment by igsbma** [comment]
@samus-aran, @SayedAlAhlMohamed 
Thank you for everything! It is good now!

**Comment by igsbma** [comment, marked as answer]
The system finally was able to get it done after another try...

**Reply by samus-aran** [reply]
Amazing! Well done 🎉 

Are you able to describe what you did differently in case that would help others with the same issue?

**Reply by igsbma** [reply]
Nothing too much differently. The Employment Verification Letter from HR is in pdf format. Since the upload only accepts JPEG, I need to open the pdf and take a screenshot. Now things become depending on your computer screen resolution. As it will limit how large you can make the screenshot. The one last time I opened the pdf on a 4k screen and the screenshot can be as large as it can.

But no matter which attempts, I always made sure it was bigger than 1024*768 as the requirement says.
